### PR TITLE
Fix group endpoint authz.

### PR DESCRIPTION
**Pull request by jgadling**
### Notes:
I broke the authz for the group membership endpoints when I switched to `row_id` -- this fixes the breakage and adds a test for multi-group-membership.

### Checklist:
- [ ] I merged latest `<base branch>`
- [ ] I manually verified the change
- [ ] I added labels to my PR
- [ ] I tested in multiple browsers
- [ ] I added relevant unit tests
- [ ] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

File: src/backend/aspen/api/authz.py [hunk 1]
```diff
@@ -219,21 +221,25 @@ def require_group_privilege(
...
     async def __call__(
         self,
-        row_id: int,
+        request: Request,
         auth_context: AuthContext = Depends(get_auth_context),
         session: AsyncSession = Depends(get_db),
     ) -> idbase:
         authz_session = AuthZSession(session, auth_context)
         query = await authz_session.authorized_query(self.privilege, self.model)
+        id_value = int(request.path_params[self.id_field])
```
Comment (lvreynoso) [inline src/backend/aspen/api/authz.py:173]: I should have caught this when reviewing, my bad.

File: src/backend/aspen/api/views/tests/test_groups.py [hunk 2]
```diff
@@ -75,6 +77,70 @@ async def test_list_members(
         assert resp_data[key] == expected[key]
 
 
+async def test_list_members_secondary_group(
```
Comment (lvreynoso) [inline src/backend/aspen/api/views/tests/test_groups.py:80]: 👍

**Review by lvreynoso — APPROVED** [review]
It looks like I broke a phylo run test that just happened to work when my PR's CI ran, I'll fix that.

File: src/backend/aspen/api/authz.py (hunk 1 above)
Comment (jgadling) [reply src/backend/aspen/api/authz.py:173]: nah, I was intentionally trying to standardize everything to `row_id` at the time but just looks like I took it toooo far.

**Pull request merged**